### PR TITLE
Add Patchage + Ganv formulae

### PR DESCRIPTION
**Pull request by ColinKinloch**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

A GUI for making connections between `jack` clients similar to `qjackctl` and `carla`.

### CODE REVIEW

File: Formula/ganv.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  def install
+    ENV.cxx11
+
+    system "./waf", "configure", "--prefix=#{prefix}"
+    system "./waf", "build"
+    system "./waf", "install"
+  end
+
+  test do
+    system "pkg-config", "ganv-1", "--libs", "--cflags"
```
Comment (fxcoudert) [inline Formula/ganv.rb]: We need a test that exercises the some of the functionality of the app.

File: Formula/patchage.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+class Patchage < Formula
+  desc "Modular patch bay for audio and MIDI systems based on Jack and Alsa"
+  homepage "https://drobilla.net/software/ganv"
+  url "https://download.drobilla.net/patchage-1.0.0.tar.bz2"
+  sha256 "6b21d74ef1b54fa62be8d6ba65ca8b61c7b6b5230cc85e093527081239bfeda9"
+  head "https://git.drobilla.net/patchage.git"
```
Comment (fxcoudert) [inline Formula/patchage.rb]: No head

File: Formula/ganv.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+class Ganv < Formula
+  desc "Gtk widget for interactive graph-like environments"
+  homepage "https://drobilla.net/software/patchage"
+  url "https://download.drobilla.net/ganv-1.4.2.tar.bz2"
+  head "https://git.drobilla.net/ganv.git"
```
Comment (fxcoudert) [inline Formula/ganv.rb]: No head

File: Formula/patchage.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  depends_on "gtkmm"
+  depends_on "jack"
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+
+    unless build.head?
+      inreplace "src/Patchage.cpp", "GtkOSXApplication",
```
Comment (fxcoudert) [inline Formula/patchage.rb]: Can we apply an upstream commit on that?

File: Formula/patchage.rb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+                                    "gtkosx_application_"
+    end
+
+    system "./waf", "configure", "--prefix=#{prefix}"
+    system "./waf", "build"
+    system "./waf", "install"
+  end
+
+  test do
+    system "#{bin}/patchage", "--help"
```
Comment (fxcoudert) [inline Formula/patchage.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/patchage.rb (hunk 4 above)
Comment (ColinKinloch) [reply Formula/patchage.rb]: It's fixed in the master branch upstream, not yet in release.

Comment (fxcoudert) [reply Formula/patchage.rb]: We could use a `patch do` block with the upstream commit.

**Pull request closed without merging**